packages/dashboard/src/dashboard-actions/dashboardReducer.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { dashboardReducer } from './dashboardReducer';
+import { onMoveAction } from './actions';
+import { Position } from '../types';
+
+describe('dashboardReducer', () => {
+  it('returns empty dashboard configuration when provided an empty dashboard', () => {
+    const x: number = 7;
```

Comment:
you can omit these explicit type declarations, they add verbosity without providing any value (as well as not matching the existing code base standards)